fineract-core/src/main/java/org/apache/fineract/commands/service/CommandWrapperBuilder.java

```diff
@@ -3416,6 +3450,15 @@ public CommandWrapperBuilder unblockSavingsAccount(final Long accountId) {
         return this;
     }
 
+    public CommandWrapperBuilder addAccrualsToSavingsAccount(final Long accountId) {
```

Comment:
what is the difference between `addAccrualsToSavingsAccount` and `excuteAccrualAccountingForSavings`?